docs/extended/gardener.md

```diff
@@ -7,4 +7,50 @@
 ### Network Policies
 If `landscape.gardener.network-policies.active` is set to `true`, garden-setup will deploy network policies into the `garden` namespace. Currently, these are only egress rules for the 'virtual' kube-apiserver and the Gardener dashboard, other components or incoming traffic are not affected for now.
 
-The default for `landscape.gardener.network-policies.active` is `false`, because the network policies have been shown to cause problems in some environments. It is planned to enable the policies by default, though, once the problems have been solved.
\ No newline at end of file
+The default for `landscape.gardener.network-policies.active` is `false`, because the network policies have been shown to cause problems in some environments. It is planned to enable the policies by default, though, once the problems have been solved.
+
+### extensions
+
+The `landscape.gardener.extensions` is optional. 
+
+#### valueOverwrites for extensions
+
+Whatever is specified in `landscape.gardener.extensions.<extensionName>.valueOverwrites` will be given directly to the helm values for the extension, so you can overwrite the corresponding default values. 
+Some values are set by default (take a look in the [deployment.yaml](../../components/gardener/extensions/deployment.yaml)), this values can also overwrite with the `landscape.gardener.extensions.<extensionName>.valueOverwrites`.
+
+The following extentions are availibe in `landscape.gardener.extensions`:
```

Comment:
typo 'extentions' instead of 'extensions'

Reply:
fixed